README.md

```diff
@@ -7,10 +7,22 @@ Static facades for using [TensorFlow](https://www.tensorflow.org/) in [ScalaPy](
 Currently contains minimal bindings for usage with TensorFlow.
 
 ## Installation
-`scalapy-tensorflow` is published on Maven Central, so you can add it to your project by including it as a dependency.
+To install `scalapy-tensorflow` use [JitPack](https://jitpack.io/)
+for scala 2.13:
+```
+scalaVersion := "2.13.3"
+
+resolvers in ThisBuild += "jitpack" at "https://jitpack.io"
+
+libraryDependencies += "com.github.VirtuslabRnD.scalapy-tensorflow" % "scalapy-tensorflow-jvm_2.13" % "develop-SNAPSHOT"
```

Comment:
Ok...  and `%%` doesn't work instead of appending `_<scala-version>`?

Reply:
i'm not sure it will include the `_jvm` part... 
Will try

Reply:
I meant sth like `"com.github.VirtuslabRnD.scalapy-tensorflow" %% "scalapy-tensorflow-jvm" % "develop-SNAPSHOT"`... so that you include the `-jvm` part yourself

Reply:
Yea sure, sorry brain lag. 
It works.